release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3400,6 +3400,43 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+{product-title} release 4.14.50, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3569[RHSA-2025:3569] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.50 --pullspecs
+----
+
+[id="ocp-4-14-50-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, installing a cluster on {vmw-first} required specifying the full path to the {vmw-short} datastore. With this release, the installation program accepts full paths and relative paths to the datastore. (link:https://issues.redhat.com/browse/OCPBUGS-54260[*OCPBUGS-54260*])
+
+* Previously, the *Cluster Settings* page would not properly render during a cluster update if the `ClusterVersion` did not receive a `Completed` update. With this release, the *Cluster Setting* page properly renders even if the `ClusterVersion` has not received a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-54167[*OCPBUGS-54167*])
+
+* Previously, incorrect addresses were being passed to the Kubernetes EndpointSlice on a cluster, and this issue prevented the installation of the MetalLB Operator on an Agent-based cluster in an IPv6 disconnected environment. With this release, a fix modifies the address evaluation method. Red{nbsp}Hat Marketplace pods can now successfully connect to the cluster API server so that the installation of MetalLB Operator and handling of ingress traffic in IPv6 disconnected environments can occur. (link:https://issues.redhat.com/browse/OCPBUGS-53314[*OCPBUGS-53314*])
+
+* Previously, when installing a cluster on {ibm-cloud-name} into an existing VPC, the installation program retrieved an unsupported VPC region. Attempting to install into a supported VPC region that follows the unsupported VPC region alphabetically caused the installation program to crash. With this release, the installation program is updated to ignore any VPC regions that are not fully available during resource lookups. (link:https://issues.redhat.com/browse/OCPBUGS-48196[*OCPBUGS-48196*])
+
+* Previously, a code migration operation failed to process external labels correctly on the *Home > Overview > Status* pane on the *Administrator perspective* of the web console. These external labels are required to prevent silenced alert notifications from getting added to *Status* pane. Because the *Status* pane did not handle external labels correctly, the pane provided links to *Alert detail* pages but generated a "no matching alerts found" message when you clicked on a link. With this release, the *Status* pane accepts external labels so clicking on an alert links to the correct *Alert detail* page. (link:https://issues.redhat.com/browse/OCPBUGS-51118[*OCPBUGS-51118*])
+
+OCPBUGS-53210? Turek
```

Comment:
Awaiting SME response for OCPBUGS-53210. I will remove this sentence tomorrow if I get no reply.